SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -384,7 +387,9 @@ const TArray<Worker_ComponentId> REQUIRED_COMPONENTS_FOR_AUTH_SERVER_INTEREST =
 								HEARTBEAT_COMPONENT_ID,
 
 								// Auth actor tag
-								ACTOR_AUTH_TAG_COMPONENT_ID
+								ACTOR_AUTH_TAG_COMPONENT_ID,
+
+								PARTITION_COMPONENT_ID
```

Comment:
Do we require here if it's in the NON_AUTH section?

Reply:
Yes I believe so because the server doesn't have authority over it, so have interest otherwise.